web_src/js/modules/stores.ts

```diff
@@ -9,3 +9,12 @@ export function diffTreeStore() {
   }
   return diffTreeStoreReactive;
 }
+
+let viewTreeStoreReactive: Reactive<Record<string, any>>;
+export function viewTreeStore() {
+  if (!viewTreeStoreReactive) {
+    viewTreeStoreReactive = reactive(window.config.pageData.viewFileInfo);
+    window.config.pageData.viewFileInfo = viewTreeStoreReactive;
+  }
+  return viewTreeStoreReactive;
+}
```

Comment:
Again, please do not just copy&paste code. Please make sure every line you write you could fully understand.

Why it needs such "store"?